data/roleDescription.js

```diff
@@ -0,0 +1,15 @@
+export const roleDescription = [
+    {
+      name: "Development",
+      description: "Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio commodi cum minus consequatur aut odio ratione sed ducimus fugit eum odit cumque. Aut tenetur animi est veniam vero est porro recusandae. Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Marketing",
+      description: "Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio commodi cum minus consequatur aut odio ratione sed ducimus fugit eum odit cumque. Aut tenetur animi est veniam vero est porro recusandae. Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Design",
+      description: "Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio commodi cum minus consequatur aut odio ratione sed ducimus fugit eum odit cumque. Aut tenetur animi est veniam vero est porro recusandae. Lorem ipsum dolor sit amet. Eum labore fugiat et distinctio Lorem ipsum dolor sit amet."
+    },
+  ];
+  
```

Comment:
Github is showing a red minus sign here because you're missing a newline. For this file specifically, it looks like your linter isn't working to indent everything consistently.

Are you using VSCode? If so, can you install the ESLint extension, and in your settings - check "Format on Save". Once this set up properly then your files will be neatly formatted and indented when you save them automatically.

Please ping me or @harsohailB if you're confused by this!

Reply:
Sure thing, I need to take advantage of VScodes extensions, hadn't realized I was missing out on this linter and Prettier.